libpromises/verify_reports.c

```diff
@@ -43,7 +43,7 @@
 #include <eval_context.h>
 #include <actuator.h>
 
-static bool PrintFile(const char *filename, size_t max_lines);
+static bool PrintFile(const char *filename, int max_lines);
```

Comment:
`ssize_t`

Reply:
But the `numlines` member – which is passed as argument – of struct `Printfile` is an int ?

Reply:
Still I think it makes more sense to change this from `size_t` to `ssize_t`.

Reply:
Alright 👍